scripts/install/opt/seagate/cortx/motr/bin/motr_setup

```diff
@@ -130,6 +130,7 @@ class TestCmd(Cmd):
     def process(self):
         sys.stdout.write(f"Processing {self.name} {self.url} {self._args}\n")
         test_lnet(self)
+	test_libfabric(self)
```

Comment:
alignment issue.

Python has very strict alignment requirement.
This will report error, I think.

Reply:
Also, this is a Python script.
Its filename, is better to be "motr_setup.py"?
Just like "motr_mini_prov.py"?

Reply:
incorporated